spec/duration.html

```diff
@@ -1586,18 +1604,39 @@ <h1>
         </dd>
       </dl>
       <emu-alg>
+        1. Let _didExpandCalendarUnit_ be *false*.
+        1. Let _date_ be ? ToTemporalDate(_isoDateTime_.[[ISODate]]).
         1. If _unit_ is ~year~, then
           1. Let _years_ be RoundNumberToIncrement(_duration_.[[Date]].[[Years]], _increment_, ~trunc~).
           1. Let _r1_ be _years_.
           1. Let _r2_ be _years_ + _increment_ × _sign_.
           1. Let _startDuration_ be ? CreateDateDurationRecord(_r1_, 0, 0, 0).
           1. Let _endDuration_ be ? CreateDateDurationRecord(_r2_, 0, 0, 0).
+          1. Let _cmpResult_ be ? CompareDurations(_endDuration_, _duration_, *undefined*, _date_, _unit_, _unit_).
+          1. If _sign_ > 0 and _cmpResult_ != 1 or _sign_ < 0 and _cmpResult_ != -1, then
+            1. Set _didExpandCalendarUnit_ to *true*.
+            1. Set _r1_ to _r2_.
+            1. Set _r2_ to _r1_ + _increment_ × _sign_.
+            1. Set _endDuration_ to ? CreateDateDurationRecord(_r2_, 0, 0, 0).
+            1. Set _cmpResult_ to ? CompareDurations(_endDuration_, _duration_, *undefined*, _date_, _unit_, _unit_).
+            1. Assert: _sign_ > 0 and _cmpResult_ is 1 or _sign_ < 0 and _cmpResult_ is -1.
+            1. Set _startDuration_ to ? CreateDateDurationRecord(_r1_, 0, 0, 0).
         1. Else if _unit_ is ~month~, then
           1. Let _months_ be RoundNumberToIncrement(_duration_.[[Date]].[[Months]], _increment_, ~trunc~).
           1. Let _r1_ be _months_.
           1. Let _r2_ be _months_ + _increment_ × _sign_.
           1. Let _startDuration_ be ? AdjustDateDurationRecord(_duration_.[[Date]], 0, 0, _r1_).
           1. Let _endDuration_ be ? AdjustDateDurationRecord(_duration_.[[Date]], 0, 0, _r2_).
+          1. Let _cmpResult_ be ? CompareDurations(_endDuration_, _duration_, *undefined*, _date_, _unit_, _unit_).
+          1. If _sign_ > 0 and _cmpResult_ != 1 or _sign_ < 0 and _cmpResult_ != -1, then
+            1. Set _didExpandCalendarUnit_ to *true*.
+            1. Set _r1_ to _r2_.
+            1. Set _r2_ to _r1_ + _increment_ × _sign_.
+            1. Set _endDuration_ to ? AdjustDateDurationRecord(_duration_.[[Date]], 0, 0, _r2_).
+            1. Set _cmpResult_ to ? CompareDurations(_endDuration_, _duration_, *undefined*, _date_, _unit_, _unit_).
+            1. Assert: _sign_ > 0 and _cmpResult_ is 1 or _sign_ < 0 and _cmpResult_ is -1.
+            1. Set _startDuration_ to ? CreateDateDurationRecord(_r1_, 0, 0, 0).
```

Comment:
This line needs to be deleted I think?

Reply:
Fixed in [4aa5f06](https://github.com/tc39/proposal-temporal/pull/3172/commits/4aa5f062635a9b03fd2c5ea7cd1ecbb7f4b43101)